### DISCUSSION TITLE
Pro: Cloning Entries - auto-populate url_title?

### DISCUSSION THREAD

**Discussion by creativearc-paul** [ExpressionEngine/ExpressionEngine#2869] [Feature Requests]
When cloning an entry using PRO, the url_title field doesn't auto-update when you are changing the title like a new entry does. Is there a way to make that happen? One of our clients uses a lot of cloning and wants to try and minimize edit time/human error as much as possible.

![clone-req](https://user-images.githubusercontent.com/224037/214681897-ec40fd52-1ab1-4b64-aee8-192597cf7683.png)

**Comment by dankingsley** [comment]
Agree, if fixed it would be great to make sure Structure url title is also included.

**Comment by intoeetive** [comment]
@creativearc-paul I doubt we can make that happen. URL titles are automatically created for new (unsaved) entries, however is case of cloning "new entry" means rather "another entry", because it is saved from the beginning.

It might be possible to add a button "update url title" which would show up when title is changed, but that might also have deeper impact on editing flow.

**Comment by chunkitdesign** [comment]
I recall that wClone add-on does have this feature
